### PR TITLE
Tutorial example: SIS spatial epidemic model

### PR DESCRIPTION
**Pull request by Song921012**
Tutorial on solving steady state of SIS (suspected-infected-suspected) reaction-diffusion  model.

### CODE REVIEW

**Review by Song921012 — COMMENTED** [review]
My first pull request, I don't know whether I do it right or not. Sorry

**Review by ChrisRackauckas — APPROVED** [review]
This looks good to me! Thanks!

File: docs/src/tutorials/sispde.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+end
+p = surface(discrete_x, discrete_t, S_solution)
+display(p)
+```
+
+### Solving steady state problem
+
+See more solvers in [Steady State Solvers · DifferentialEquations.jl](https://diffeq.sciml.ai/stable/solvers/steady_state_solve/)
+
+```julia
```
Comment (ChrisRackauckas) [inline docs/src/tutorials/sispde.md:104]: why remove the doctesting?

Comment (Song921012) [reply docs/src/tutorials/sispde.md:104]: Hello Chris, Project.toml only includes OrdinaryDiffEq, document.yml is not successful.

Comment (ChrisRackauckas) [reply docs/src/tutorials/sispde.md:104]: Add it.

Comment (Song921012) [reply docs/src/tutorials/sispde.md:104]: Ok, Chris. Thank you.

**Review by ChrisRackauckas — APPROVED** [review]

**Comment by ChrisRackauckas** [comment]
@xtalax there are test failures on master? That's not good.

**Comment by xtalax** [comment]
The CI was passing on last merge, the fail was on nonuniform right? There are some rand calls on those tests, some of those values may push the tests outside of tolerance. I can hard code some suitable values if that is preferable.

**Comment by xtalax** [comment]
@Song921012 please push a dummy commit to verify this

**Comment by Song921012** [comment]
> 



>

**Comment by Song921012** [comment]
Sorry, wrong click! I will dummy commit. @xtalax

**Pull request merged**